File: KingdomCodeTool/wwwroot/Download/202501201119/Angular/shared/Sys_UserCommands.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Sys_UserCommands } from './Sys_UserCommands.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Sys_UserCommandsService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'User_Id', 'Command_Id', 'Function_Id', 'Menu_Id'];

    List: Sys_UserCommands[] | undefined;
    ListFilter: Sys_UserCommands[] | undefined;
    FormData!: Sys_UserCommands;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Sys_UserCommands";
    }
}

File: KingdomCodeTool/wwwroot/Download/202501201119/Service/Implement/Sys_UserCommandsService.cs
namespace Service_eHospital_DongNai_A_System.Implement
{
    public class Sys_UserCommandsService : BaseService<Sys_UserCommands, ISys_UserCommandsRepository>
    , ISys_UserCommandsService
    {
    private readonly ISys_UserCommandsRepository _Sys_UserCommandsRepository;
    public Sys_UserCommandsService(ISys_UserCommandsRepository Sys_UserCommandsRepository) : base(Sys_UserCommandsRepository)
    {
    _Sys_UserCommandsRepository = Sys_UserCommandsRepository;
    }
    }
    }

